build.gradle

```diff
@@ -96,7 +95,8 @@ dependencies {
     implementation group: 'org.simpleframework', name: 'simple-xml', version: '2.7.1'
     implementation group: 'net.javacrumbs.shedlock', name: 'shedlock-spring', version: '4.21.0'
     implementation group: 'net.javacrumbs.shedlock', name: 'shedlock-provider-jdbc-template', version: '4.21.0'
-    runtimeOnly group: 'com.github.ben-manes.caffeine', name: 'jcache', version: '2.8.8'
+    implementation group: 'com.github.ben-manes.caffeine', name: 'jcache', version: '2.8.8'
```

Comment:
why was it necessary to change this?

Reply:
reverted this (was experimenting with multiple cache managers before and is not required anymore)